server/aws-lsp-codewhisperer/src/language-server/chat/chatController.ts

```diff
@@ -73,7 +60,9 @@ export class ChatController implements ChatHandlers {
         let response: GenerateAssistantResponseCommandOutput
 
         try {
+            this.#log('Request from tab:', params.tabId, JSON.stringify(requestInput.data))
             response = await session.generateAssistantResponse(requestInput.data)
+            this.#log('Response to tab:', params.tabId, JSON.stringify(response.$metadata))
```

Comment:
Do we want to keep this verbose logging or are these temporary log statements?

Reply:
vscode has something similar in their debug log. But for now, I trimmed down on the input logging to include only conversation id. We can revisit on this at a later point.